### ISSUE TITLE
Convert "+1" comments to a "like" automatically

### ISSUE THREAD

**Issue by andreynering** [dear-github/dear-github#152]
Now that GitHub allows "liking" an issue/PR, it would be really useful to covert comments like `+1` and `:+1:` to a like instead.

I never tried, but seems that GitLab does this.

Most people still comment "+1" instead of clicking on :+1: button, which causes a lot of noise for watchers.

**Comment by koenpunt** [comment]
This maybe also have to do with the fact that reactions can't be added from the mobile website

**Comment by andreynering** [comment]
> This maybe also have to do with the fact that reactions can't be added from the mobile website

Right. There are actually many features not available in the mobile site, which could be a separated issue.

I forgot to mention it could work for the other available emojis, too, like :-1: , :heart:, etc.

**Comment by TPS** [comment]
@andreynering @koenpunt Just FYI, https://github.com/isaacs/github/issues/141 & https://github.com/isaacs/github/issues/640 address your comments above.… ☺

**Comment by arshaw** [comment]
I posted this one https://github.com/isaacs/github/issues/640 and will also post it here:

-----

I've created an alternative UI for browsing and ranking Github Issues. See my newly created project:

[ghi-scoreboard](https://github.com/arshaw/ghi-scoreboard) (Github project)
[example scoreboard](http://fullcalendar.io/issues/) (fullcalendar)
[blog announcement](http://fullcalendar.io/blog/2016/07/renovated-issue-tracker/)

One of its features is the ability to use these lame +1 comments for ranking purposes (albeit aggregated on the server-side, periodically). If you set `aggregateComments:true`, can you sort by `plusComments` (+1 or 👍  within a comment) or `plusScore`, which combines comments with real reactions.

Hope it comes in handy for some of you.

**Comment by dessant** [comment]
I've made a GitHub App that deletes +1 comments, and encourages people to use reactions instead. :tada: 

https://probot.github.io/apps/reaction/

I think auto-conversion would not be viable, because it cannot be guessed if a user is reacting to a recent comment, or the issue in general.

**Comment by lukehefson** [comment]
Hello everyone 👋. I'm a Product Manager at GitHub working on the [Papercuts initiative](https://blog.github.com/2018-08-28-announcing-paper-cuts/), and as part of the project we've just shipped: https://blog.github.com/changelog/2018-09-24-hiding-repeat-comments/

So, it's not the same solution as the one proposed here, but I hope that it should go some way to make the problem of threads which are choked up with multiple repeat `+1`s being a little easier to read.  We will likely iterate on this in the future. In the meantime – if you want to _completely_ delete `+1`s – I'd definitely suggest installing @dessant's excellent App [referenced above](https://github.com/dear-github/dear-github/issues/152#issuecomment-401445843)!

For the problem of encouraging people not to post **_more_** `+1`s – that's something we're also investigating!

**Comment by Tikotik** [comment]
> For the problem of encouraging people not to post **_more_** `+1`s – that's something we're also investigating!

As in this issue #https://github.com/isaacs/github/issues/283
It seems most people comment just because 

> Commenting +1 so I can use is:issue commenter:<myusername> until view list of issues I'm subscribed to on the website is solved`

**Comment by sbusch** [comment]
+1

**Comment by SergioDRivas** [comment]
+1

**Comment by CT-66** [comment]
+1

**Comment by SandeepKS21** [comment]
+1
